catalogs/catalog-lakehouse-hudi/src/main/java/org/apache/gravitino/catalog/hudi/backend/hms/HudiHMSSchema.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.catalog.hudi.backend.hms;
+
+import org.apache.gravitino.catalog.hudi.HudiSchema;
+import org.apache.hadoop.hive.metastore.api.Database;
+
+public class HudiHMSSchema extends HudiSchema {
```

Comment:
Why do we need to define different Schema for different backend?

Reply:
Since Hudi doesn't provide a unified Schema class or interface, I defined different HudiSchema to represent the properties from different backends to preserve their unique properties. I also defined an abstract class `org.apache.gravitino.catalog.lakehouse.hudi.HudiSchema.Builder<T>` for interconverting HudiSchema and backendSchema in different HudiSchema.